scripts/us_census/pep/monthly_population/download.py

```diff
@@ -0,0 +1,153 @@
+""" A Script to download USA Census PEP monthly population data
+    from the URLS in provided json file.
+    Typical usage:
+    1. python3 download.py
+    2. python3 download.py -f file_urls.json
+    3. python3 download.py -f <<path_to_file>>
+"""
+import os
+import json
+import argparse
+import pandas as pd
+import numpy as np
+
+HEADER = 1
+SKIP_ROWS = 1
+
+
+def save_data(url, download_local_path):

```

Comment:
Pls add data types to the arguments and return value:

`def save_data(url: str, download_local_path: str):`

Reply:
Sure Ajai.